sdk/monitor/monitor-query/src/models/constants.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Common ISO8601 durations.
+ */
+export const CommonDurations = {
+  /** Alias for ISO8601 value 'P7D' */
+  last7Days: "P7D",
+  /** Alias for ISO8601 value 'P3D' */
+  last3Days: "P3D",
+  /** Alias for ISO8601 value 'P2D' */
+  last2Days: "P2D",
+  /** Alias for ISO8601 value 'P1D' */
+  lastDay: "P1D",
+
+  /** Alias for ISO8601 value 'PT1H' */
+  lastHour: "PT1H",
+  /** Alias for ISO8601 value 'PT4H' */
+  last4Hours: "PT4H",
+  /** Alias for ISO8601 value 'P1D' */
+  last24Hours: "P1D",
+  /** Alias for ISO8601 value 'P2D' */
+  last48Hours: "P2D",
+
+  /** Alias for ISO8601 value 'PT30M' */
+  last30Minutes: "PT30M",
+  /** Alias for ISO8601 value 'PT5M' */
+  last5Minutes: "PT5M"
+};
```

Comment:
These constants won't change will they? Right now the type info makes it look like all these values are just strings, but you can have them be string literals if you add `as const` to the end of your assignment:

```typescript
const CommonDurations = {
/* fields */
} as const;
```

Reply:
Oh, interesting. I will make that fix, didn't know about that...but now I remember @witemple-msft doing something similar in dev-tool.

Reply:
Done